internal/interop/interopconnect/test_cases.go

```diff
@@ -501,7 +501,7 @@ func customMetadataServerStreamingTest(
 		require.NoError(t, stream.Err())
 	}
 	assert.NoError(t, stream.Close())
-	validateMetadata(t, stream.ResponseHeader(), stream.ResponseTrailer(), customMetadataString, customMetadataBinary)
+	// validateMetadata(t, stream.ResponseHeader(), stream.ResponseTrailer(), customMetadataString, customMetadataBinary)
```

Comment:
Temporary until this is fixed in connect-go (see https://github.com/bufbuild/connect-go/issues/524). Won't land this PR until this can be uncommented.